src/components/graphics/PlaceholderEmpty/PlaceholderEmpty.tsx

```diff
@@ -0,0 +1,104 @@
+
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
+|* distributed with this file, you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react';
+import { classNames as cx, type ComponentProps } from '../../../util/componentUtil.ts';
+import { assertUnreachable } from '../../../util/types.ts';
+
+import { Icon } from '../Icon/Icon.tsx';
+
+import cl from './PlaceholderEmpty.module.scss';
+
+export { cl as PlaceholderEmptyClassNames };
+
+export type PlaceholderEmptyIconType = 'graph' | 'folder' | 'file' | 'file-error';
+
+export type PlaceholderEmptySize = 'large' | 'small';
+
+export type PlaceholderEmptyProps = React.PropsWithChildren<ComponentProps<'div'> & {
+  /** Whether this component should be unstyled. */
+  unstyled?: undefined | boolean,
+  
+  /** A size of this component. */
+  size?: undefined | PlaceholderEmptySize,
+
+  /** An icon type of this component. */
+  iconType?: undefined | PlaceholderEmptyIconType,
+  
+  /** A custom icon of this component. */
+  customIcon?: undefined | React.ReactNode,
+
+  /** A title of this component. Mandatory. */
+  title: string,
+  
+  /** A sub message to be displayed next to the title. */
+  subtitle?: undefined | string | React.ReactNode,
+  
+  /** Any additional actions to be shown in this component, such as a button. */
+  actions?: undefined | React.ReactNode,
+}>;
+/**
+ * A way to display the absence of data.
+ */
+export const PlaceholderEmpty = (props: PlaceholderEmptyProps) => {
+  const {
+    unstyled = false,
+    size = 'large',
+    iconType = 'graph',
+    customIcon = null,
+    title = '',
+    subtitle = '',
+    actions = null,
+    children,
+    ...propsRest
+  } = props;
+  
+  const icon = ((): PlaceholderEmptyIconType => {
+    switch (iconType) {
```

Comment:
shouldn't the component be re-usable with any available icon?

Reply:
In Figma, it looks like these 4 icons will be built-in images, so I defined `PlaceholderEmptyIconType` to make this clear, similar to the icons in `Alert.tsx`. Users can also customize it through the `customIcon` prop. Please let me know if you have any better suggestions.

Reply:
IMHO, the icons that are present in the Figma are just example use cases, I don't think we need some sort of pre-defined set of default icons to choose from. 
I think we just need an icon of type IconName, 
and an option to pass a custom icon (in case consumer want to use some other type of graphic, png for example etc..)